Src/ReleaseHistory.md

```diff
@@ -8,6 +8,7 @@
 - SDK: Sarif.PatternMatcher projects will start using a fixed version of `RE2.Managed` and `Strings.Interop`. [#638](https://github.com/microsoft/sarif-pattern-matcher/pull/638)
 - Bump Microsoft.Security.Utilities from 1.1.0 to 1.3.0. [#642](https://github.com/microsoft/sarif-pattern-matcher/pull/642)
 - BUG: Fix false negative when `SearchSkimmer` is invoked directly and `MaxFileSizeInKilobytes` is not set. This will now default to approximately 10MB. [#637](https://github.com/microsoft/sarif-pattern-matcher/pull/637)
+- Dependency alignment of `Newtonsoft.Json` to 12.0.3. [#644](https://github.com/microsoft/sarif-pattern-matcher/pull/644)
```

Comment:
>Dependency alignment of `Newtonsoft.Json` to 12.0.3. [](http://example.com/codeflow?start=2&length=52)

Is this dependency alignment visible to someone using spam? Are there customer use cases where they would be integrating directly with the Sarif.PatternMatcher.dll as opposed to the CLI?
Also, this suggests that the CLI is also downgraded to 12.0.3 as written.